test/functional/feature_xfield_maxblocksize.py

```diff
@@ -99,7 +100,43 @@ def set_test_params(self):
     def reconnect_p2p(self, node):
         node.disconnect_p2ps()
         node.add_p2p_connection(TestP2PConn(self.nodes[0].time_to_connect))
-        node.p2p.wait_for_getheaders(timeout=5)
+        node.p2p.wait_for_getheaders(timeout=TAPYRUSD_MIN_TIMEOUT)
+
+    def wait_for_node_ready(self, node_index, expected_blocks=None, timeout=TAPYRUSD_REORG_TIMEOUT):
```

Comment:
There is a similar method in `feature_federation_management.py` but can't they be unified?